docs/_ug/commandSummary/itemCommands/set.md

```diff
@@ -1,2 +0,0 @@
-<!-- markdownlint-disable-file first-line-h1 -->
-<b>set id/INDEX_LIST [n/ITEM_NAME] [qty/QUANTITY] [unit/UNIT] [bgt/BOUGHT_DATE] [exp/EXPIRY_DATE]</b><br><br>❗ IMPORTANT:<br>Do not update multiple items to have the same name<br><br><u>Valid Examples:</u><br>set id/1 n/Potatoes qty/60 unit/kg<br>set id/1,2,3 qty/60
```

Comment:
Is the "❗ IMPORTANT:<br>Do not update multiple items to have the same name" not going to be transferred over to `edit.md`?

Reply:
I think this should be an implementation detail, not a warning shown to the user. I'll open this as an issue actually.

Reply:
I'll confirm this with Yi Xian first, as he refactored Edit command from AB3!